src/main/java/org/zalando/nakadi/service/EventPublisher.java

```diff
@@ -145,6 +151,11 @@ private void validateSchema(final JSONObject event, final EventType eventType) t
         }
     }
 
+    private void validateEventSize(final BatchItem item) throws EventValidationException {
+        if (item.getEventSize() > nakadiSettings.getEventMaxBytes())
```

Comment:
We better add a rule in the code style for enforcing braces usage. I think it's the preferred style over the code base. While we don't do it, could you please add braces here?